docs-samples/V4/JS/manage-conversation-and-user-state/file-storage.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * Botbuilder v4 SDK - Add suggested actions
```

Comment:
should be "File Storage" … but since filestorage is going away, I don't think we need this sample any more....